### ISSUE TITLE
Having issues after latest update

### ISSUE THREAD

**Issue by tommiekn** [teamtnt/laravel-scout-tntsearch-driver#274]
Hi,

Im getting this from a composer update, running laravel 6:

```
> @php artisan package:discover

   Symfony\Component\Debug\Exception\FatalThrowableError  : syntax error, unexpected 'private' (T_PRIVATE)

  at vendor/teamtnt/laravel-scout-tntsearch-driver/src/Console/StatusCommand.php:113
```

**Comment by nticaric** [comment]
please try again, but before running composer cacheclear

**Comment by tommiekn** [comment]
@nticaric same issue also after a 

`composer clearcache`

**Comment by nticaric** [comment]
just published a new release v8.3.0 this should solve the issue

**Comment by tommiekn** [comment]
Worked, thanks!
